packages/bson-transpilers/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "bson-transpilers",
+  "version": "0.15.0",
+  "apiVersion": "0.0.1",
+  "productName": "BSON Transpilers",
+  "description": "Source to source compilers using ANTLR",
+  "contributors": [
+    "Anna Herlihy <herlihyap@gmail.com>",
+    "Alena Khineika <alena.khineika@mongodb.com>",
+    "Irina Shestak <irina.shestak@mongodb.com>"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "prepare": "npm run compile",
+    "precompile": "node download-antlr.js",
+    "compile": "npm run antlr4-js && npm run antlr4-py && npm run symbol-table",
+    "antlr4-js": "java -Xmx500M -cp './antlr-4.7.2-complete.jar:$CLASSPATH' org.antlr.v4.Tool  -Dlanguage=JavaScript -lib grammars -o lib/antlr -visitor -Xexact-output-dir grammars/ECMAScript.g4",
+    "antlr4-py": "java -Xmx500M -cp './antlr-4.7.2-complete.jar:$CLASSPATH' org.antlr.v4.Tool  -Dlanguage=JavaScript -lib grammars -o lib/antlr -visitor -Xexact-output-dir grammars/Python3.g4",
+    "symbol-table": "node compile-symbol-table.js",
+    "test": "mocha",
+    "prepublishOnly": "npm run compile",
+    "depcheck": "depcheck",
+    "precheck": "npm run depcheck",
+    "check": "mongodb-js-precommit './codegeneration/**/*{.js,.jsx}' './test/**/*.js' index.js"
```

Comment:
Need to remove precommit before merging, it messes up local dev environment

Reply:
This one (mongodb-js-precommit) or `pre-commit`?

Reply:
`mongodb-js-precommit` yeah (internally it has a dependency on pre-commit)